config/install/field.field.node.publication.field_publication_authors.yml

```diff
@@ -1,16 +1,19 @@
+uuid: 53e1711e-9e06-497f-8b29-7361d0bd0273
```

Comment:
please don't add uuid for the config/install. This uuid represents the unique id for a specific site, so installing it for different project will create issues.

Reply:
Thanks @MdNadimHossain , my bad, I didn't realise it was added it. I have removed them. Thanks.